### ISSUE TITLE
Quiet mode fails

### ISSUE THREAD

**Issue by wjketting** [arthurkrupa/gree-hvac-mqtt-bridge#21]
Hi there,

Gree-hvac-mqtt-bridge works fine, exept when putting the airco in quiet mode.
The next error message pops up in the app log:
![afbeelding](https://user-images.githubusercontent.com/46381645/51087565-45125880-1755-11e9-9bc9-227448774668.png)
curious to know whether it's me doing something wrong or a bug in het bridge?

**Comment by arthurkrupa** [comment]
It looks like the properties returned by your HVAC don't include 'Quiet'. This feature was added by a PR, so I'm not sure if this works at all (maybe the property code is wrong?).

Are you able to run the code locally on your computer and debug? If so, add the following just before line 38 in index.js:

```
console.log(JSON.stringify(this.device.props, null, 4));
```

**Comment by wjketting** [comment]
Arthur,

thanks for your respons.
I'm running home assistant as hassio install on a raspberry pi 3b. Not sure how to find your file to modify it. Directory is empty when opening in config webinterface (see below).
Bridge stops running btw if I put the airco in quiet mode with error message as stated above.

btw I love the bridge, works like a charm. Thanks for all the work you have put into it.

![afbeelding](https://user-images.githubusercontent.com/46381645/51130384-2da69e80-182d-11e9-9ce0-199b0ecffb56.png)

**Comment by wjketting** [comment]
pressed the wrong button when commenting... ;)

**Comment by arthurkrupa** [comment]
In order to debug, you have to stop the addon on your RPi Hass.io and run the code directly on your computer:

1. Download the code of this repo to your computer (either by `git clone` or unpack the ZIP)
2. Open index.js and add `console.log(JSON.stringify(this.device.props, null, 4));` before line 38.
3. Install NodeJS (if you don't have it)
4. Open the terminal (Command Line on Windows), go to the folder with the code, run `npm install` (this installs dependencies)
5. Still in the terminal, run the bridge with:

```
node index.js --hvac-host="YOUR_HVAC_IP" --mqtt-broker-url="mqtt://YOUR_HASSIO_IP" --mqtt-topic-prefix="home/greehvac" --mqtt-username="YOUR_MQTT_USERNAME" --mqtt-password="YOUR_MQTT_PASSWORD"
```

6. If your HVAC is connected, turn on "quiet mode" using your remote
7. Look inside the terminal if a JSON-like structure was returned just before the error. Paste the content of this JSON.

**Comment by wjketting** [comment]
Been experimenting a bit after above error.
After changing "console.log(JSON.stringify(this.device.props, null, 4));" to "console.log(JSON.stringify(deviceModel.props, null, 4));" running the bridge in NodeJS worked fine. After enabling the "quiet" mode on the airco (by remote control) next was reported in the console.
![afbeelding](https://user-images.githubusercontent.com/46381645/51200046-ac1c4280-18f8-11e9-8443-75f054b6992e.png)
Hope this helps finding the solution to prevent the bridge from crashing.

**Comment by wjketting** [comment]
when exploring further I found next values for swingHor:
```
            full: 1, // swing in full range
            fixedLeft: 2, // fixed in leftmost position (1/5)
            fixedMidLeft: 3, // fixed in middle-left postion (2/5)
            fixedMid: 4, // fixed in middle position (3/5)
            fixedMidRight: 5, // fixed in middle-right postion (4/5)
            fixedRight: 6, // fixed in rightmost position (5/5)
```
I also have a 7 but not sure yet what mode exactly corresponds to that number (seems same as 1)

**Comment by wjketting** [comment]
Based on the above (and some minor other stuff I discovered while playing with the code) did some changes on the bridge, tested it and made a pull request to updated the addon.

**Comment by arthurkrupa** [comment]
Closed by #22 . Thank you for the fix!
